include/barrett/bus/abstract/communications_bus.h

```diff
@@ -45,7 +45,7 @@ namespace bus {
 class CommunicationsBus {
 public:
 	static const size_t MAX_MESSAGE_LEN = 8;  /** The maximum of any of the available communications buses */
-	static const double TIMEOUT = 1.0;  /** Bus connection timeout limit in seconds */
+	static const double TIMEOUT = 0.01;  /** Bus connection timeout limit in seconds */
```

Comment:
At some point should we think a bit about whether we are minimally fixing this library with the intent of merging them upstream, or just changing anything we think is wrong for our system.

It's possible this timeout has some purpose in being long on some other system configuration.  It's  also not really related to the issue at hand.